src/studio/nuxeo-studio/nuxeo-studio-release-notes.md

```diff
@@ -12,54 +12,29 @@ tree_item_index: 730
 Discover the changes we brought in our latest Nuxeo Studio updates.
 
 
```

Comment:
```suggestion
# January
```

Reply:
See comments about month displayed twice in a page.

Reply:
All the changes are down